cloudinit/userdata_cloudinit.erb

```diff
@@ -41,6 +49,28 @@ users:
    lock-passwd: false
    passwd: <%= @host.root_pass %>
 
+ runcmd:
+ <% if rhel_compatible -%>
+ - <%= snippet('epel') -%>
```

Comment:
XD, I know everyone has an opinion on indentation etc.  It will probably go back and forth.
I wish there was a good linter for yaml to enforce a standard indentation like the ```puppet-lint-strict_indent-check``` gem and put it to rest.

I will probably just put this one in an indent block to be consistent.  There is always the possibility that someone adds another line to the epel template later.  That would make it more robust.